### DISCUSSION TITLE
What is the minimum support Linux kernel version？

### DISCUSSION THREAD

**Discussion by yafeiaa** [florianl/go-conntrack#33] [General]
Hi, have a problem：
What is the minimum support Linux kernel version?

**Comment by florianl** [comment]
hi
Moving this question to [Discussions](https://github.com/florianl/go-conntrack/discussions) as it is not a issue with the go package but a general question.
As far as I can tell, the first parts of conntrack were merged with commit [080774a2](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=080774a243f56ce2195ace96fba3d18548ee48ce) and so are available since Linux kernel v2.6.14. In subsequent commits more attributes were added to conntrack.

**Reply by yafeiaa** [reply]
I understand, thank you very much ~
